apps/public-docsite-v9/.storybook/main.js

```diff
@@ -55,7 +55,7 @@ module.exports = /** @type {Omit<import('../../../.storybook/main'), 'typescript
     charts: {
       title: 'Charts v9',
       // Workaround to enable docsite using PR workflow till master workflow is enabled
-      url: 'https://fluentuipr.z22.web.core.windows.net/pull/33270/chart-docsite/storybook',
+      url: 'https://fluentuipr.z22.web.core.windows.net/pull/33270/chart-docsite/storybook/',
```

Comment:
This url was a workaround. Can we replace this with this url https://storybooks.fluentui.dev/charts/